### PR TITLE
fix glog format error in node_container_manager.go

### PR DESCRIPTION
**Pull request by dixudx**
**What this PR does / why we need it**:

Fix glog format in node_container_manager.go

**Which issue this PR fixes** :

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Review by resouer — APPROVED** [review]
/lgtm

**Comment by dixudx** [comment]
@resouer waiting for your approval. Thx.

**Pull request closed without merging**
